src/libNode/Node.h

```diff
@@ -239,10 +239,10 @@ class Node : public Executable, public Broadcastable
     bool ProcessFinalBlock(const std::vector<unsigned char>& message,
                            unsigned int offset, const Peer& from);
     bool ProcessForwardTransaction(const std::vector<unsigned char>& message,
-                                   unsigned int offset, const Peer& from);
+                                   unsigned int cur_offset, const Peer& from);
     bool
     ProcessForwardTransactionCore(const std::vector<unsigned char>& message,
-                                  unsigned int offset);
+                                  unsigned int cur_offset);
```

Comment:
Although it is automated, I prefer it to be without _ for consistency with other code. same for block_num.